exporters/powerdns/exporter.yml

```diff
@@ -14,8 +14,10 @@ exporter_commit: fffbd7c4768681f93988c2c3287b690db20e6ce0
 exporter_changelog: https://github.com/lotusnoir/prometheus-powerdns_exporter/tags
 # Enable packages for Linux
 package_linux: true
+# Set goarch values
+package_linux_goarchs: amd64,arm64
 # Enable packages for Windows
-package_windows: true
+package_windows: false
```

Comment:
Just an NIT, if we are not packaging wor Windows, we might want to remove all the GUIDs below...

Just to remove noise on the file...

Reply:
+1

Reply:
Thanks for pointing it out, I removed them here: [726b401](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/148/commits/726b401d1b5c468ecfdd3d4a95507b48c4ac0ecb)

Reply:
I should remove as well the `build-exporter-windows.sh`

Reply:
Good catch! I included a comment to clarify the script is kept for testing purposes instead of removing it: [108e4a0](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/148/commits/108e4a0b3431dbaa27bfe0daaa277f19a6d5235b) WDYT?